### PR TITLE
update libretro packages

### PR DESCRIPTION
**Pull request by ToKe79**

### CODE REVIEW

**Comment by Ntemis** [comment]
Cant accept what you did to ppsspp package, fix it and i can merge

File: packages/libretro/ppsspp/package.mk [hunk 1]
```diff
@@ -19,7 +19,7 @@
 ################################################################################
 
 PKG_NAME="ppsspp"
-PKG_VERSION="bebc63c"
+PKG_VERSION="abee4eb"
```
Comment (RobLoach) [inline packages/libretro/ppsspp/package.mk]: @Ntemis  What's wrong with https://github.com/libretro/ppsspp/pull/28 ?

Comment (Ntemis) [reply packages/libretro/ppsspp/package.mk]: OMG sorry i was working on aarch64 asm opts for this core and got distracted :)
Fixed my comment and thanks for the pointing out

Comment (Ntemis) [reply packages/libretro/ppsspp/package.mk]: fixed by editing my comment

**Comment by ToKe79** [comment]
@Ntemis: should I add a patch to change in deps/libretro/Makefile on lines 22-23:
ROOT_DIR := ../..
CORE_DIR := ../..
to
ROOT_DIR := .
CORE_DIR := .
Would that be accepted?
Thanks.

**Comment by Ntemis** [comment]
Just remove those dir changes, why you messed with them in the first place?
You run make in dep/libretro but .so is in root so you never go back to it so am curious why it didnt failed on your part. :)

**Comment by ToKe79** [comment]
the .so was then in deps/libretro, so it did not fail

**Comment by ToKe79** [comment]
Done as discussed on IRC, also corrected the package.mk of sameboy same way. Thanks for advices/help.

**Comment by Ntemis** [comment]
Still dont like what you did to sameboy package sorry

**Comment by RobLoach** [comment]
Updated it to take a similar pattern to what we use elsewhere:
https://github.com/ToKe79/Lakka-LibreELEC/blob/libretro-update/packages/libretro/sameboy/package.mk#L37-L43

``` make
configure_target() {
  cd $PKG_BUILD
}

make_target() {
  make -C libretro
}
```

**Comment by Kivutar** [comment]
Closed by demand of @ToKe79

**Pull request closed without merging**